packages/cli/src/template/interface.ts.ejs

```diff
@@ -0,0 +1,12 @@
+// Copyright 2020-2021 OnFinality Limited authors & contributors
```

Comment:
we can remove license header for code template. We may own the copyright of template, but not the code generated from the template.